Dockerfile_ams_clearlinux

```diff
@@ -1,14 +1,58 @@
+FROM clearlinux as base_build
+
```

Comment:
Could you add a comment that we build here tensorflow-serving-api  pip package for python3.8 and TFS version 2.0
The container could be named pip build for example.